modules/govuk_jenkins/templates/jobs/bouncer_cdn_redirect.yaml.erb

```diff
@@ -0,0 +1,64 @@
+---
+- scm:
+    name: cdn-configs_Bouncer_CDN
+    scm:
+        - git:
+            url: git@github.com:alphagov/cdn-configs.git
+            branches:
+              - master
+
+- job:
+    name: Bouncer_CDN
+    display-name: Bouncer_CDN
+    project-type: freestyle
+    description: |
+        <a href='https://deploy.blue.production.govuk.digital/job/Bouncer_CDN/'> 
+            <img src='https://www.futurefinance.com/img/blogimages/2015/12/We_have_moved1.png'
+        </a>
+        <p style=''>
+            <strong>So we have moved to a new location click on the image or link to get redirected to the correct Jenkins instance.
```

Comment:
I don't think we need an image for this (and the above may be copyrighted, we do not want futurefinance.com to know when we open this page, etc...)
Maybe rather something like:
```
<strong>
This job has been migrated to AWS. Please execute it from 
<a href='https://deploy.blue.production.govuk.digital/job/Bouncer_CDN/'>
https://deploy.blue.production.govuk.digital/job/Bouncer_CDN/
</a>
</strong>
```